MAUI/DataGrid/conditional-styling.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: Conditional Styling in .NET MAUI DataGrid control | Syncfusion
+description: Learn all about Conditional Styling support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control, its elements and more.
+platform: MAUI
+control: SfDataGrid
+documentation: ug
+---
+
+# Conditional Styling in .NET MAUI DataGrid (SfDataGrid)
+The [SfDataGrid] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.html) allows to customize the style of the individual cells and rows based on the requirements. It can be customized in the following ways:
+
+1. Using column CellStyle
+2. Using Converter
+3. Using DataTriggers
+
+## Conditional Row Style
+The record rows ([DataGridRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridRow.html)) can be customized conditionally by changing its property value based on ‘cell value’ or ‘data object’ by using converter, where converter returns the value based on underlying record.
```

Comment:
@FarjanaParveen  - 

The data rows can be customized conditionally by writing the style with converter for [DataGridRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridRow.html) control. Its `BindingContext` is underlying datasource object.